src/Catalog/Helpers/Utils.cs

```diff
@@ -44,38 +51,94 @@ public static string GetResource(string resName)
 
             return _resourceStringCache[resName];
         }
-
-        public static IGraph CreateNuspecGraph(XDocument nuspec, string baseAddress)
+        
+        public static IGraph CreateNuspecGraph(XDocument nuspec, string baseAddress, bool normalizeXml = false)
         {
-            nuspec = NormalizeNuspecNamespace(nuspec);
-
-            XslCompiledTransform transform = CreateTransform("xslt.nuspec.xslt");
-
             XsltArgumentList arguments = new XsltArgumentList();
-            arguments.AddParam("base", "", baseAddress + "packages/");
+            arguments.AddParam("base", "", baseAddress);
```

Comment:
See NuSpecStage.cs - just moving "packages/" from helper to caller. This was necessary to combine duplicate code (Utils.CreateNuSpecGraph, and CatalogItem.CreateNuSpecGraph).